### PR TITLE
Issue 1510/views rename to lists

### LINKED ISSUE #1510: Rename "views" to "lists"
## Description
We have decided to rename the "views" feature to lists. It should happen everywhere in the UI.

## Relevant Job Stories
None

## Estimated size (S, M, L)
I think this issue is medium because it's mostly search and replace, but in a lot of places.

## Prerequisites
None

## Requirements
- [x] Change all English (default) strings that mention views to use "Lists" instead. Only change the user-visible text, not the names of the strings or the message prefix
- [x] Update URLs
- [ ] Change identifiers in code (source directories, variables, etc) – low priority

## Possible implementations
N/A

## Design specifications
None

## Open questions
None

### PR DESCRIPTION
**Pull request by rebecarubio**
## Description
This PR renames "Views" for "Lists" in all user-visible places and updates the urls. It also fixes the tests affected by the change.

## Screenshots
image
image
image
image

## Changes
* Changes localize strings that are visible 
* Changes `pages/views` folder for `pages/lists `
* Changes user-visible urls to "/lists"
* Fixes integration tests to follow the change.

## Notes to reviewer
I didn't push the autogenerated yaml file `src/locale/en.yml `. I can do it if that's necessary.

## Related issues
Resolves #1510

### CODE REVIEW

**Review by ziggabyte — CHANGES_REQUESTED** [review]
There are some places where list is written with upper case "L" inside a sentence. I think it looks weird. I think all these places should change to "list". The option is to change all the other occurrances to "List", but that I think should also mean consistently changing the first letter of *all* our features to uppercase, everywhere that is user-facing. Basically: consistent use of capitalization is what I want.

Some places i found "views" in the wild: 
In call assignments, I think "sheet" here should be replaced with list.
bild

This is in smart search filter gallery
bild
this is the texts relate to that filter
bild
...an the sentence in the representation of all filters
bild

**Comment by rebecarubio** [comment]
> There are some places where list is written with upper case "L" inside a sentence. I think it looks weird. I think all these places should change to "list". The option is to change all the other occurrances to "List", but that I think should also mean consistently changing the first letter of _all_ our features to uppercase, everywhere that is user-facing. Basically: consistent use of capitalization is what I want.
> 
> Some places i found "views" in the wild: In call assignments, I think "sheet" here should be replaced with list. bild
> 
> This is in smart search filter gallery bild this is the texts relate to that filter bild ...an the sentence in the representation of all filters bild

Thanks for the great review! I'll agree with the changes, so I'll implement them :)

**Review by ziggabyte — APPROVED** [review]
Looking goooood! So happy to finally let go of those upper case letters that have been itching my eyes 🎃

File: src/features/views/l10n/messageIds.ts [hunk 1]
```diff
@@ -2,46 +2,46 @@ import { ReactElement } from 'react';
 
 import { m, makeMessages } from 'core/i18n';
 
-export default makeMessages('feat.views', {
+export default makeMessages('feat.lists', {
```
Comment (richardolsson) [inline src/features/views/l10n/messageIds.ts:5]: This should never have been renamed. It breaks all translations. Can someone please undo this, either @kaulfield23 or @ziggabyte and submit a new PR?

Comment (ziggabyte) [reply src/features/views/l10n/messageIds.ts:5]: I'll do it!

**Pull request merged**